build.sh

```diff
@@ -0,0 +1,49 @@
+function isinstalled {
+  if command -v "$1";
+  then
+    true
+  else
+    false
+  fi
+}
+
+# doing this to make this work locally since 99% of the times most dev machines already have curl and might not have apt-get
+if isinstalled "curl"
+then
+  echo "curl already installed moving on"
+else
+  apt-get update && apt install -y curl
+fi
+
+NANCY_VERSION=v0.1.15
```

Comment:
Suggestion: Might be neat to make it check if `NANCY_VERSION` is empty and then set it, that way you could default it, but then allow someone to have this set in CI and pull a different one? I dunno, not totally needed, just an idea

Reply:
Something like this?? d29730d

Reply:
And then i can do something like this even locally which is nice. 
```NANCY_VERSION=v0.1.13 ./build.sh```

Reply:
^^^ yeah, totally!